src/FileCatalog/ComplexNumbers/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+
+namespace ComplexNumbers
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Program program = new Program();
+
+            Console.WriteLine("Enter a");
+            double a = program.ChangedLine();
+            Console.WriteLine("Enter b");
+            string b = Convert.ToString(program.ChangedLine()) + "i";
+            ComplexNumbers complexNumber = new ComplexNumbers(a, b);
+            complexNumber.NumToString();
+            const string addControl = "A";
+            const string magnitudeControl = "M";
+            const string stringControl = "S";
+            const string exitControl = "E";
+
+            bool isExit = false;
+            string input;
+
+            while (!isExit)
+            {
+                input = Console.ReadLine();
+                switch (input)
+                {
+                    case magnitudeControl:
+                        {
+                            complexNumber.GetMagnitude();
+                            break;
+                        }
+                    case stringControl:
+                        {
+                            complexNumber.NumToString();
+                            break;
+                        }
+                    case addControl:
+                        {
+                            Console.WriteLine("Enter real part of complex number");
+                            double a1 = program.ChangedLine();
+                            Console.WriteLine("Enter imaginary part of complex number");
+                            double b1 = program.ChangedLine();
+                            string bi1 = Convert.ToString(b1) + "i";
+                            complexNumber.Add(a1, bi1);
+                            break;
+                        }
+                    case exitControl:
+                        {
+                            isExit = true; break;
+                        }
+                    default: Instuctions(); break;
+
+
+                }
+            }
+        }
+        public double ChangedLine()
```

Comment:
This method is necessary to check the user input values. It belongs to the class ComplexNumber testing organization and is in the Program class